src/pre_defined_classes/simple_programs_taking_numeric_user_input/medium/q7/MainTest.java

```diff
@@ -0,0 +1,47 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.medium.q7;
+
+
+/*
+Sample output:
+Enter your score:
+-5
+Your goal is 45. Keep up the good work!
+ */
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Random;
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+    //Parsons
+    public Clause[] testSentence(){
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "-5";
+        return new Clause[]{
+                new StringLiteral("Enter your score: "),
+                new NewLine(),
+                new StringLiteral("Your goal is "),
+                new IntegerLiteral("goal"),
```

Comment:
This is where you can use RandomInteger, which tells the framework to check if this number is randomly distributed

Reply:
If I used RandomInteger(), then we will encounter the problem of having to define the upper bound of the random number. To resolve that issue I changed the question so that the input has a range.